examples/viewports.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * viewports.c
```

Comment:
For a better user experience, I suggest to use this example with Sway or a similar  tiling Wayland-based window manager.